pkg/storaged/client.js

```diff
@@ -91,13 +91,27 @@ define([
     /* D-Bus proxies
      */
 
+    var STORAGED_SERVICE =   "org.storaged.Storaged";
+    var STORAGED_OPATH_PFX = "/org/storaged/Storaged";
+    var STORAGED_IFACE_PFX = "org.storaged.Storaged";
+
+    /* This might happen eventually.
+     */
+    if (false) {
+       STORAGED_SERVICE =   "org.freedesktop.UDisks2";
+       STORAGED_OPATH_PFX = "/org/freedesktop/UDisks2";
+       STORAGED_IFACE_PFX = "org.freedesktop.UDisks2";
+    }
+
     client.time_offset = undefined;  /* Number of milliseconds that the server is ahead of us. */
-    client.storaged_features = undefined;
+    client.features = undefined;
 
-    client.storaged_client = cockpit.dbus("org.storaged.Storaged");
+    client.storaged_client = cockpit.dbus(STORAGED_SERVICE);
 
     function proxy(iface, path) {
-        return client.storaged_client.proxy(iface, path, { watch: true });
+        return client.storaged_client.proxy(STORAGED_IFACE_PFX + "." + iface,
```

Comment:
same question as with proxies: should we rename the function if we add an explicit prefix?